### ISSUE TITLE
TSLint expect trailing comma on multiline function args in React

### ISSUE THREAD

**Issue by brunolm** [palantir/tslint#2474]
### Bug Report

- __TSLint version__: 4.5.1
- __TypeScript version__: 2.2.2
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
<div className="members">
  { members.map(member =>
    <img key={ member.id } src={ member.img } />
  ) }
</div>
```

with `tslint.json` configuration:

```json
{
  "extends": [
    "tslint:latest"
  ]
}
```

which adds

```json
    "trailing-comma": [
      true,
      {
        "multiline": "always",
        "singleline": "never"
      }
    ]
```

#### Actual behavior
`[tslint] Missing trailing comma (trailing-comma)` after img tag

#### Expected behavior
No errors

Is there a way to add exceptions for cases like this?

**Comment by andy-hanson** [comment]
A simpler example:
```ts
[1, 2, 3].map(x =>
    x + 1 // Missing trailing comma
);
```

Not sure what the rule for an exception would be. Personally I would just format that as:

```ts
<div className="members">
  {
    members.map(member =>
      <img key={ member.id } src={ member.img } />);
  }
</div>
```

**Comment by brunolm** [comment]
Cool, that works (without the semicolon there). Thank you.

I was initially thinking of something like:

```
"trailing-comma": [
  true,
  {
    "multiline": "always",
    "singleline": "never",
    "jsx": "never"
  }
]
```

I don't know how complex that would be and if it would be worth implementing that.

**Comment by andy-hanson** [comment]
It doesn't look like this is really about jsx though. See my first example above.
